### PR TITLE
cleaned up a lot of language, added links to AMP Conf videos…

### PR DESCRIPTION
**Pull request by CrystalOnScript**
...included info on SXG.

Resolves issue [#2828](https://github.com/ampproject/amp.dev/issues/2828)

### CODE REVIEW

File: pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/publishing_checklist.md [hunk 1]
```diff
@@ -9,41 +9,49 @@ contributors:
...
 ---
 
-Follow this checklist to ensure your pages are getting all the benefits AMP has to offer by being healthy and accessible on the web! 
-
+Follow this checklist to give your site the fullest AMP experience! 
 
 # Ensure AMP Specification Validation 
 
-AMP comes with a ton of built in benefits, such as decreasing content load times by preloading content from AMP Caches. However, only documents that can be verified as valid AMP get these benefits. If you publish documents that have critical AMP errors your pages will not be indexed by an AMP cache, or possible served as error pages. 
+AMP comes with a ton of built in benefits, such as decreasing user wait time by preloading content from AMP Caches. To get these benefits, pages must be valid AMP documents. Pages published with critical AMP errors are not indexable by AMP Caches, and possibly served as error pages.
```
Comment (sebastianbenz) [inline pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/publishing_checklist.md]: nit: rephrase `critical AMP errors` to `errors reported by the AMP validator`

File: pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/publishing_checklist.md [hunk 2]
```diff
@@ -9,41 +9,49 @@ contributors:
...
 
-Learn more about the tools you can use to validate your AMP pages:
+Never publish an invalid AMP page again by using these tools:
 
+* [Validate AMP pages](../../../documentation/guides-and-tutorials/learn/validation-workflow/validate_amp.md?format=websites)
+* [The AMP Validator ](https://validator.ampproject.org/)
+* [Google AMP Tester](https://search.google.com/test/amp)
+* [AMP Tools](../../../documentation/tools.html?format=websites)
```
Comment (sebastianbenz) [inline pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/publishing_checklist.md]: Add amp-linter https://github.com/ampproject/amp-toolbox/tree/master/packages/linter

File: pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/publishing_checklist.md [hunk 3]
```diff
@@ -9,41 +9,49 @@ contributors:
...
 
 
-*   [Validate AMP pages](../../../documentation/guides-and-tutorials/learn/validation-workflow/validate_amp.md?format=websites)
-*   [The AMP Validator ](https://validator.ampproject.org/)
-*   [Google AMP Tester](https://search.google.com/test/amp)
-*   [AMP Tools](../../../documentation/tools.html?format=websites) 
+# Grant cached AMP pages server access
 
+Great news, valid AMP pages automatically opt into all existing AMP Caches! This means your users experience content that loads efficiently and safely. These types of optimizations are great, but come with a small catch.
+Some users will be served AMP pages from domains that do not match your own. This can cause pages to lose access to site data when using dynamic AMP components such as [`<amp-form>`](../../../documentation/components/reference/amp-form.md?format=websites) or [`<amp-list>`](../../../documentation/components/reference/amp-list.md?format=websites). These types of errors are Cross-Origin Resource Sharing, or CORS, issues. Work with safety, not against it, by enabling CORS Requests from all available [AMP Caches](https://cdn.ampproject.org/caches.json)!
```
Comment (sebastianbenz) [inline pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/publishing_checklist.md]: Add: `If you're using Node.js in your backend, you can use the [amp-cors middleware](https://github.com/ampproject/amp-toolbox/tree/master/packages/cors).`

File: pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/publishing_checklist.md [hunk 4]
```diff
@@ -9,41 +9,49 @@ contributors:
...
-*   [The AMP Validator ](https://validator.ampproject.org/)
-*   [Google AMP Tester](https://search.google.com/test/amp)
-*   [AMP Tools](../../../documentation/tools.html?format=websites) 
+# Grant cached AMP pages server access
 
+Great news, valid AMP pages automatically opt into all existing AMP Caches! This means your users experience content that loads efficiently and safely. These types of optimizations are great, but come with a small catch.
+Some users will be served AMP pages from domains that do not match your own. This can cause pages to lose access to site data when using dynamic AMP components such as [`<amp-form>`](../../../documentation/components/reference/amp-form.md?format=websites) or [`<amp-list>`](../../../documentation/components/reference/amp-list.md?format=websites). These types of errors are Cross-Origin Resource Sharing, or CORS, issues. Work with safety, not against it, by enabling CORS Requests from all available [AMP Caches](https://cdn.ampproject.org/caches.json)! 
 
-# Grant cached AMP documents server access
+Learn more about accessing server data:
```
Comment (sebastianbenz) [inline pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/publishing_checklist.md]: What does server data mean?

File: pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/publishing_checklist.md [hunk 5]
```diff
@@ -9,41 +9,49 @@ contributors:
...
+* [How AMP Pages are cached ](../../../documentation/guides-and-tutorials/learn/amp-caches-and-cors/how_amp_pages_are_cached.md?format=websites)
+* [CORS in AMP](../../../documentation/guides-and-tutorials/learn/amp-caches-and-cors/amp-cors-requests.md?format=websites)
+* [AMP CORS Library](https://www.npmjs.com/package/amp-toolbox-cors)
```
Comment (sebastianbenz) [inline pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/publishing_checklist.md]: remove in favor of sentence above

File: pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/publishing_checklist.md [hunk 6]
```diff
@@ -9,41 +9,49 @@ contributors:
...
 
-Learn more about accessing server data:
 
+# Shareable, safe, and substantiated content with signed exchanges
```
Comment (sebastianbenz) [inline pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/publishing_checklist.md]: what does `substantiated content` mean?

File: pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/publishing_checklist.md [hunk 7]
```diff
@@ -9,41 +9,49 @@ contributors:
...
+* [How AMP Pages are cached ](../../../documentation/guides-and-tutorials/learn/amp-caches-and-cors/how_amp_pages_are_cached.md?format=websites)
+* [CORS in AMP](../../../documentation/guides-and-tutorials/learn/amp-caches-and-cors/amp-cors-requests.md?format=websites)
+* [AMP CORS Library](https://www.npmjs.com/package/amp-toolbox-cors)
 
-Learn more about accessing server data:
 
+# Shareable, safe, and substantiated content with signed exchanges 
 
+Keep your domain's URL while maintaining portable content by implementing signed exchanges (SXG). By serving AMP pages with SXG, digital signatures protect your information by tying the document to its claimed URL. Implementing SXG delivers signed AMP content in addition to, rather than instead of, regular AMP content.
```
Comment (sebastianbenz) [inline pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/publishing_checklist.md]: what does `maintaining portable content ` mean? Maybe mention the benefit of easier analytics and user sessions as cookies are always first party.

File: pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/publishing_checklist.md [hunk 8]
```diff
@@ -54,16 +62,19 @@ When loading your AMP pages in an AMP Cache, check via your [browser’s develop
...
-*   [AMP on Google, Google AMP Cache](https://developers.google.com/amp/cache/overview)]
-*   [Debug AMP Cache issues](../../../documentation/guides-and-tutorials/learn/amp-caches-and-cors/amp-cache-debugging.md?format=websites)
-*   [AMP Cache URL Format and Request Handling](../../../documentation/guides-and-tutorials/learn/amp-caches-and-cors/amp-cache-urls.md?format=websites)
+* [Using the Google AMP Cache](../../../documentation/examples/documentation/Using_the_Google_AMP_Cache.html?format=websites)
+* [AMP on Google, Google AMP Cache](https://developers.google.com/amp/cache/overview)
+* [Debug AMP Cache issues](../../../documentation/guides-and-tutorials/learn/amp-caches-and-cors/amp-cache-debugging.md?format=websites)
+* [AMP Cache URL Format and Request Handling](../../../documentation/guides-and-tutorials/learn/amp-caches-and-cors/amp-cache-urls.md?format=websites)
+
+
+# Destined for discoverability
```
Comment (sebastianbenz) [inline pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/publishing_checklist.md]: Better than the original version, but I still have no idea what this means ;-)

File: pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/publishing_checklist.md [hunk 9]
```diff
@@ -54,16 +62,19 @@ When loading your AMP pages in an AMP Cache, check via your [browser’s develop
...
 
-# Set up discoverable canonicals 
+AMP first and paired AMP pages all need to ensure they are discoverable! All AMP pages required the `<link rel="canonical" href="$SOME_URL">` in their `<head>`. AMP first pages need to link to themselves and AMP pages paired to a non-AMP page will need to link link to each other.
```
Comment (sebastianbenz) [inline pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/publishing_checklist.md]: Maybe explain AMP First and paired AMP.

File: pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/publishing_checklist.md (hunk 8 above)
Comment (sebastianbenz) [reply pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/publishing_checklist.md]: Maybe: `Ensure your AMP files are discoverable by search engines`

File: pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/publishing_checklist.md [hunk 10]
```diff
@@ -88,24 +99,19 @@ X-Robots-Tag: noindex                      <= don't!
...
-
-*   [So your AMP test doesn't perform — now what?](https://blog.amp.dev/2018/11/08/so-your-amp-test-doesnt-perform%e2%80%8a-%e2%80%8anow-what/)
-*   [Cache vs. non-cache analysis](https://support.google.com/analytics/answer/6343176?hl=en#cache)
-*   [Measuring user journeys across the AMP Cache and your website](https://blog.amp.dev/2018/11/08/so-your-amp-test-doesnt-perform%e2%80%8a-%e2%80%8anow-what/)
-
- 
+* [So your AMP test doesn't perform — now what?](https://blog.amp.dev/2018/11/08/so-your-amp-test-doesnt-perform%e2%80%8a-%e2%80%8anow-what/)
+* [Cache vs. non-cache analysis](https://support.google.com/analytics/answer/6343176?hl=en#cache)
+* [Measuring user journeys across the AMP Cache and your website](https://blog.amp.dev/2018/11/08/so-your-amp-test-doesnt-perform%e2%80%8a-%e2%80%8anow-what/)
+* [Measuring success: What's new in AMP analytics & experiments (AMP Conf '19)](https://www.youtube.com/watch?v=wPW-kXsONqA&list=PLXTOW_XMsIDSY0USlzgoaIkRyPcHklrEl&index=27)
```
Comment (sebastianbenz) [inline pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/publishing_checklist.md]: Maybe add a note here that SXG can greatly simplify analytics.

File: pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/publishing_checklist.md (hunk 6 above)
Comment (CrystalOnScript) [reply pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/publishing_checklist.md]: I'm saying in the sense that the content is, in fact, from the source. But I'll just remove it from the heading.

File: pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/publishing_checklist.md (hunk 8 above)
Comment (CrystalOnScript) [reply pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/publishing_checklist.md]: perfect, I'll take it.

**Comment by CrystalOnScript** [comment]
Updated per nits @sebastianbenz :)

**Pull request merged**